### DISCUSSION TITLE
[Configuration Help] Not clear on how to add HTTPS

### DISCUSSION THREAD

**Discussion by sTiKyt** [firefly-iii/firefly-iii#8048] [General]
I have my Firefly III server running fine, after precisely following steps described in "Using Docker Compose" section at
https://docs.firefly-iii.org/firefly-iii/installation/docker/

I can access my firefly instance by going to specific IP through web browser

But how do i attach it to something like subdomain.domain.com?

I've tried figuring it out online, but after many steps, and some outdated guide i only managed to mess up and break my instance, had to delete it and reinstall everything again.

Is there any clear step-by-step instruction on how to attach this instance to a subdomain, add SSL, etc.?

Could someone explain it to me please?

**Comment by JC5** [comment]
Do you already have a domain or a subdomain available?

**Reply by sTiKyt** [reply]
Sure

**Reply by sTiKyt** [reply]
So, any way to achieve my requirements?

**Reply by JC5** [reply]
Sure, where do you want me to begin? I can answer specific questions but I'm not a tech support oracle.
